devnets/1620/shared/genesis.json

```diff
@@ -0,0 +1,842 @@
+{
+  "config": {
+    "chainId": 1620,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "mergeForkBlock": 0,
+    "terminalTotalDifficulty": 0,
+    "shanghaiTime": 1683680926
+  },
+  "number": "0x0",
+  "nonce": "0x1",
+  "difficulty": "0x1",
+  "gasLimit": "0x280de80",
+  "gasUsed": "0x0",
+  "timestamp": "0x645AAFA0",
```

Comment:
0x645AA640

Reply:
1683898800+2400 is `0x645E4B10`. Updated.